.golangci.yaml

```diff
@@ -9,6 +9,8 @@ linters:
     - dupl
     - gosimple
     - staticcheck
+    - nlreturn
+    - misspell
```

Comment:
Does this not complain about things like `kubectl`, `k3s`, `rke2`, etc.?

Reply:
apparently no lol

Reply:
Thank goodness!